install/k8s/README.md

```diff
@@ -0,0 +1,49 @@
+# Kubernetes Setup for Contiv
+
+This document details the setup instructions for Kubernetes version 1.4 and higher
+
+### Setup Kubernetes Cluster
```

Comment:
Do we have way to install user entered version of k8s (1.4+) ? Its good to have this variable flexible in the script.